spinta/types/type.py

```diff
@@ -235,6 +241,33 @@ def load_type(context: Context, prop: Node, data: dict, manifest: Manifest):
     return load(context, type, data, manifest)
 
 
+@load.register()
+def load(context: Context, type: Type, value: object) -> object:
+    # loads value to python native value according to given type
+    return type.load(value)
+
+
+@load.register()
+def load(context: Context, type: Array, value: object) -> object:
+    # loads value into native python list, including all list items
+    array_item_type = type.items.type
+    loaded_array = type.load(value)
+    for j, item in enumerate(loaded_array):
+        # overwrite loaded list, with items loaded to native python types
+        loaded_array[j] = load(context, array_item_type, item)
```

Comment:
I'm not sure if this is a good idea, to update input value in place. Lets append loaded values to a new array and leave input array as is.